doc/ddu-filter-converter_devicon.txt

```diff
@@ -64,6 +91,18 @@ defaultIconHlgroup	(string)
 
 	Default: ""
 
+			 *ddu-filter-converter_devicon-param-specificFileIcons*
+specificFileIcons	(dictionary)
+	Sets the icon applied to a file with a specific name.
+
+	Default: {}
+
+			       *ddu-filter-converter_devicon-param-customIcons*
+customIcons	(dictionary)
+	Manually specify an icon for each extension.
+
+	Default: {}
+
```

Comment:
I think this is not very friendly, and I would like you to explain in writing what the values of key and value are, so that I can understand them without looking at the examples.